app/src/main/res/layout/fragment_product_detail.xml

```diff
@@ -262,35 +275,33 @@
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
                 android:layout_alignParentBottom="true"
-                android:background="@color/color_white"
-                android:orientation="horizontal"
-                android:padding="@dimen/dp_3"
+                android:background="@color/color_gray_white"
                 >
-                <ImageView
-                    android:id="@+id/image_user"
-                    android:layout_width="@dimen/dp_0"
-                    android:layout_height="@dimen/dp_45"
-                    android:layout_marginRight="@dimen/dp_3"
-                    android:layout_weight="1.3"
-                    android:padding="@dimen/dp_1"
-                    app:imageUrl="@{viewModel.userImage}"
-                    />
-                <EditText
-                    android:layout_width="@dimen/dp_0"
+                <LinearLayout
+                    android:layout_width="match_parent"
                     android:layout_height="wrap_content"
-                    android:layout_weight="7"
-                    android:hint="@string/add_a_comment"
-                    android:text="@={viewModel.comment}"
-                    />
-                <ImageButton
-                    android:layout_width="@dimen/dp_0"
-                    android:layout_height="@dimen/dp_35"
-                    android:layout_marginLeft="@dimen/dp_3"
-                    android:layout_weight="1"
-                    android:background="@drawable/ic_send_gray"
-                    android:onClick="@{() -> viewModel.onClickSendComment()}"
-                    />
-
+                    android:layout_alignParentBottom="true"
+                    android:layout_margin="@dimen/dp_7"
+                    android:background="@drawable/border_comment"
+                    android:orientation="horizontal"
+                    android:padding="@dimen/dp_3"
+                    >
+                    <EditText
+                        android:layout_width="@dimen/dp_0"
+                        android:layout_height="wrap_content"
+                        android:layout_weight="9"
+                        android:hint="@string/add_a_comment"
+                        android:text="@={viewModel.comment}"
+                        />
+                    <ImageButton
+                        android:layout_width="@dimen/dp_0"
+                        android:layout_height="@dimen/dp_40"
+                        android:layout_marginLeft="@dimen/dp_3"
```

Comment:
WARNING: [RtlHardcoded] Consider adding `android:layout_marginStart="@dimen/dp_3"` to better support right-to-left layouts
 Using `Gravity#LEFT` and `Gravity#RIGHT` can lead to problems when a layout is rendered in locales where text flows from right to left. Use `Gravity#START` and `Gravity#END` instead. Similarly, in XML `gravity` and `layout_gravity` attributes, use `start` rather than `left`.

For XML attributes such as paddingLeft and `layout_marginLeft`, use `paddingStart` and `layout_marginStart`. *NOTE*: If your `minSdkVersion` is less than 17, you should add *both* the older left/right attributes *as well as* the new start/right attributes. On older platforms, where RTL is not supported and the start/right attributes are unknown and therefore ignored, you need the older left/right attributes. There is a separate lint check which catches that type of error.

(Note: For `Gravity#LEFT` and `Gravity#START`, you can use these constants even when targeting older platforms, because the `start` bitmask is a superset of the `left` bitmask. Therefore, you can use `gravity="start"` rather than `gravity="left|start"`.)